src/Globe/TileMesh.js

```diff
@@ -80,7 +84,7 @@ function TileMesh(params, builder, geometryCache) {
     }
 
     // Layer
-    this.currentElevation = -1;
+    this.currentElevation = NO_ELEVATION;
```

Comment:
> Texture's Elevation has null, it's not sufficient to check if there's no elevation?

Same comment
It's not possible to remove `this.currentElevation`?
or remove one state `-1` or `NO_ELEVATION`?
Maybe we need add level in elevation texture, like image texture.